### PR TITLE
create stub package for utilities

### LINKED ISSUE #24625: @fluentui/utilities (re-exports @uifabric/utilities)

### PR DESCRIPTION
**Pull request by micahgodbolt**
fixes #24625

### CODE REVIEW

**Comment by micahgodbolt** [comment]
needs a change file still

**Comment by fabricteam** [comment]
## [Perf Analysis](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">7.0 Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/BaseButton-mount_base.html">916</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/BaseButton-mount.html">925</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Breadcrumb-mount_base.html">42732</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Breadcrumb-mount.html">42564</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Checkbox-mount_base.html">1534</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Checkbox-mount.html">1558</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/CheckboxBase-mount_base.html">1295</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/CheckboxBase-mount.html">1296</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/ChoiceGroup-mount_base.html">5080</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/ChoiceGroup-mount.html">5066</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/ComboBox-mount_base.html">893</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/ComboBox-mount.html">917</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/CommandBar-mount_base.html">8048</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/CommandBar-mount.html">8014</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/ContextualMenu-mount_base.html">14114</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/ContextualMenu-mount.html">13978</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/DefaultButton-mount_base.html">1146</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/DefaultButton-mount.html">1170</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/DetailsRow-mount_base.html">3812</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/DetailsRow-mount.html">3790</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/DetailsRowFast-mount_base.html">3749</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/DetailsRowFast-mount.html">3714</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/DetailsRowNoStyles-mount_base.html">3613</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/DetailsRowNoStyles-mount.html">3569</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Dialog-mount_base.html">2266</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Dialog-mount.html">2245</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/DocumentCardTitle-mount_base.html">1835</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/DocumentCardTitle-mount.html">1833</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Dropdown-mount_base.html">2593</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Dropdown-mount.html">2590</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/FocusTrapZone-mount_base.html">1739</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/FocusTrapZone-mount.html">1730</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/FocusZone-mount_base.html">1811</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/FocusZone-mount.html">1895</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/IconButton-mount_base.html">1819</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/IconButton-mount.html">1745</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Label-mount_base.html">334</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Label-mount.html">350</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Layer-mount_base.html">1951</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Layer-mount.html">1927</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Link-mount_base.html">437</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Link-mount.html">466</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/MenuButton-mount_base.html">1482</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/MenuButton-mount.html">1467</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/MessageBar-mount_base.html">2096</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/MessageBar-mount.html">2178</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Nav-mount_base.html">3395</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Nav-mount.html">3301</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/OverflowSet-mount_base.html">1381</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/OverflowSet-mount.html">1396</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Panel-mount_base.html">1483</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Panel-mount.html">1494</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Persona-mount_base.html">972</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Persona-mount.html">949</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Pivot-mount_base.html">1505</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Pivot-mount.html">1472</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/PrimaryButton-mount_base.html">1326</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/PrimaryButton-mount.html">1311</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Rating-mount_base.html">7914</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Rating-mount.html">7806</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/SearchBox-mount_base.html">1302</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/SearchBox-mount.html">1302</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Shimmer-mount_base.html">2683</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Shimmer-mount.html">2612</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Slider-mount_base.html">1501</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Slider-mount.html">1513</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/SpinButton-mount_base.html">5148</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/SpinButton-mount.html">5181</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Spinner-mount_base.html">420</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Spinner-mount.html">423</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/SplitButton-mount_base.html">3202</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/SplitButton-mount.html">3167</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Stack-mount_base.html">509</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Stack-mount.html">498</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/StackWithIntrinsicChildren-mount_base.html">1691</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/StackWithIntrinsicChildren-mount.html">1727</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/StackWithTextChildren-mount_base.html">4779</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/StackWithTextChildren-mount.html">4912</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/SwatchColorPicker-mount_base.html">10663</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/SwatchColorPicker-mount.html">10675</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/TagPicker-mount_base.html">2888</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/TagPicker-mount.html">2859</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/TeachingBubble-mount_base.html">52064</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/TeachingBubble-mount.html">51598</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Text-mount_base.html">431</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Text-mount.html">433</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/TextField-mount_base.html">1378</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/TextField-mount.html">1411</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Toggle-mount_base.html">837</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/Toggle-mount.html">812</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/button-mount_base.html">109</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24627/perf-test/button-mount.html">102</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

File: stub-packages/utilities/CHANGELOG.json [hunk 1]
```diff
@@ -0,0 +1,4677 @@
+{
+  "name": "@fluentui/utilities",
+  "entries": [
+    {
```
Comment (GeoffCoxMSFT) [inline stub-packages/utilities/CHANGELOG.json:4]: I think we want to clear out the changelog for stub packages, right?

**Review by GeoffCoxMSFT — APPROVED** [review]

File: stub-packages/utilities/CHANGELOG.json (hunk 1 above)
Comment (micahgodbolt) [reply stub-packages/utilities/CHANGELOG.json:4]: yeah, i wasn't sure what the expectations were there

**Pull request merged**